tests/capture/test_capture_transforms.py

```diff
@@ -199,3 +202,255 @@ def g(i):
 
         assert qfunc_jaxpr.eqns[1].primitive == qml.Z._primitive
         assert qfunc_jaxpr.eqns[2].primitive == qml.measurements.ExpectationMP._obs_primitive
+
+
+# pylint: disable=protected-access, expression-not-assigned
+class TestMapWiresTransform:
+    """Unit tests for the MapWiresTransform class."""
+
+    def test_qnode_simple(self):
+        """Test that a simple qnode is transformed correctly."""
+
+        const = 0.1
```

Comment:
```suggestion
        const = jax.numpy.array(0.1)
```
If the `const` is `0.1`, I think it gets hardcoded into the jaxpr rather than being treated as a closure variable.